### ISSUE TITLE
Data Loader >> Update does not provide any logs

### ISSUE THREAD

**Issue by hpreets** [xjsender/haoide#109]
I updated existing records using Data Loader >> Update. Although in the Status Bar it showed success, but the records were not updated correctly into Salesforce. I cannot see any error logs, not even in the Panel >> Open Error Panel.

I used the same file to upload using Salesforce Import Wizard and it went through successfully. My sample file containing just two columns, one being Salesforce Id and other being a Text Area field. File size 57KB and 134 as total number of rows.

**Comment by xjsender** [comment]
@hpreets , the log is saved in the same folder with your provided CSV file.

**Comment by xjsender** [comment]
@hpreets , did you check my reply for your issue? thanks.

**Comment by hpreets** [comment]
Yeah, I just checked the source folder containing the CSV but did not find the log file. Let me try again and I will update this thread. Thanks :-)

**Comment by xjsender** [comment]
@hpreets , if the status bar tell you it's succeed but no log file found at the same folder with the CSV file, you should check the sublime console, maybe there are some exception happen.

**Comment by hpreets** [comment]
This is what I see in Sublime Text console

```
[Debug for new_job]:
<?xml version="1.0" encoding="UTF-8"?>
<jobInfo xmlns="http://www.force.com/2009/06/asyncapi/dataload">
    <operation>update</operation>
    <object>GurbaniTuk__c</object>
    <concurrencyMode>Parallel</concurrencyMode>
    <contentType>CSV</contentType>
</jobInfo>
```
```
[Debug for close_job]: 
<?xml version="1.0" encoding="UTF-8"?>
<jobInfo xmlns="http://www.force.com/2009/06/asyncapi/dataload">
    <state>Closed</state>
</jobInfo>
```
Also Log Panel and Error Panel under HaoIDE >> Panel are blank.

**Comment by xjsender** [comment]
@hpreets , didn't you find the log file at the same folder with your csv file?

**Comment by hpreets** [comment]
No @xjsender I did not find any logs in the same folder with my csv.

**Comment by lushang** [comment]
No response for years
